src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/not-available.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import { I18n } from '@ngx-translate/i18n-polyfill';
+
+@Pipe({
+  name: 'notAvailable'
+})
+export class NotAvailablePipe implements PipeTransform {
```

Comment:
This pipe looks more or less equal to the already existing `empty` pipe, maybe it's because the naming is a little bit misleading. Maybe you can refactor and rename the `empty` pipe.

Reply:
Maybe `defaultTo` would be a good name.

Reply:
I see a different use case for each of them.
- `EmptyPipe` is used do indicate that no value was set for that column (by the user)
- `NotAvailablePipe` is used when ceph dashboard cannot calculate the value for that column

Having a single `defaultTo` pipe for both scenarios will require to pass a parameter to the pipe (e.g.: `-` or `n/a)`, making it harder to use in tables because we would be forced to specify a template for the cell (because datatable column spec `pipe` does not support parameters).

Reply:
@votdev and @ricardoasmarques so I guess this can be marked as resolved?